addons/google_recaptcha/models/res_config_settings.py

```diff
@@ -1,11 +1,13 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import fields, models
+from odoo import api, fields, models
+from odoo.tools.misc import str2bool
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
+    enable_recaptcha = fields.Boolean("Enable reCAPTCHA", config_parameter='enable_recaptcha', groups='base.group_system')
```

Comment:
You are adding this field but I don't see you removing the existing field which trigger the uninstallation, is that normal?

Reply:
See: https://github.com/odoo/odoo/pull/178166#discussion_r1928885511 :+1: